cordova-plugin-outline/apple/src/OutlinePlugin.swift

```diff
@@ -182,18 +180,17 @@ class OutlinePlugin: CDVPlugin {
     } else {
       uuid = NSUUID().uuidString
     }
-    event.message = "\(OutlinePlugin.kPlatform) report (\(uuid))"
+    event.message = SentryMessage(formatted: "\(OutlinePlugin.kPlatform) report (\(uuid))")
 
     OutlineSentryLogger.sharedInstance.addVpnExtensionLogsToSentry()
-    Client.shared?.send(event: event) { (error) in
-      if error == nil {
-        self.sendSuccess(true, callbackId: command.callbackId)
-        Client.shared?.breadcrumbs.clear() // Breadcrumbs are persisted, clear on send success.
-      } else {
-        self.sendError("Failed to report event: \(String(describing: error))",
-                       callbackId: command.callbackId)
-      }
+    // As of sentry-cocoa 5.x, it's no longer possible to set a callback when sending an event,
+    // which means that we cannot determine whether the event has been sent successfully.
+    // See: https://github.com/getsentry/sentry-cocoa/issues/660
```

Comment:
This is a little worrying.  Let's at least change the error to that the event failed to queue, which is what's happening with this check according to https://docs.sentry.io/platforms/apple/migration/

Reply:
> This is a little worrying.

I agree. The issue I linked does say that the event will be stored on network failures.


> Let's at least change the error to that the event failed to queue [...]

For our purpose, the efffect of failing to queue the event is the same as failing to report it.